src/components/List/ListGroupItem.react.js

```diff
@@ -28,7 +30,7 @@ function ListGroupItem({
   );
   const Component = RootComponent || "a";
   return (
-    <Component className={classes}>
+    <Component href={href} className={classes}>
```

Comment:
Can you take a look at the new more flow friendly, react-router supporting, way we're trying to use RootComponent in [NavLink](https://github.com/tabler/tabler-react/blob/master/src/components/Nav/NavLink.react.js) and implement here?